beets/util/__init__.py

```diff
@@ -949,12 +949,21 @@ def interactive_open(targets, command):
     """
     assert command
 
-    # Split the command string into its arguments.
-    try:
-        args = shlex.split(command)
-    except ValueError:  # Malformed shell tokens.
+    if os.name == 'nt':
         args = [command]
 
+        if command == 'cmd.exe':
+            targets = ['/C start ' + targets[0]]
```

Comment:
You're silently dropping any `targets` beyond the first here; that seems like a bad idea. A brief `grep` suggests that we're not using `targets` lists with more than one items, but the signature of `itneractive_open` currently suggests that this is a supported thing. Since, according to https://github.com/beetbox/beets/issues/2847#issuecomment-375395370, opening several files at once is probably not going to work on Windows, we might want to change that. However, I would prefer to actually change the function signature to take a single filename instead of a list.